### ISSUE TITLE
Buffer Share Broken with Hyphens

### ISSUE THREAD

**Issue by brianleejackson** [warfare-plugins/social-warfare#155]
With the new Buffer share button it breaks when sharing a WordPress post with a hyphen in the title... it outputs it as this: `&#8211;`

Thanks

**Comment by DustinWStout** [comment]
Hey @brianleejackson! We're trying to recreate this on our test site and are unsuccessful so far. Can you give me the exact title you're seeing this happen to?

**Comment by brianleejackson** [comment]
Just re-enabled Buffer button for a bit so you can see. [https://woorkup.com/social-warfare-review/](https://woorkup.com/social-warfare-review/) Steps to replicate below:

**Step 1** 
Scroll down and click on buffer on left hand side.

![image](https://cloud.githubusercontent.com/assets/186682/15261524/eb248f1c-1911-11e6-962d-b425e415e4f6.png)

Buffer pops up and you can see URL is messed up. I am in latest version of Chrome.

![image](https://cloud.githubusercontent.com/assets/186682/15261516/db1a607e-1911-11e6-9b37-a63279bf719b.png)

Thanks

**Comment by brianleejackson** [comment]
Any updates on this? Haven't been able to use Buffer sharing yet. Thanks!

**Comment by Nicholas-Cardot** [comment]
@brianleejackson I'll try to figure out what's happening on this by Tuesday. I reached out to Buffer the other day to try to find documentation about their share URL, but the person running the Twitter account didn't seem to know much about what I was trying to ask. We're going to send them an email and hopefully we can get a better response that way.

**Comment by Nicholas-Cardot** [comment]
Actually, I think I may have just realized what is causing this to happen! I'm going to go test something right now!

**Comment by Nicholas-Cardot** [comment]
@brianleejackson Can you install and test this version and let me know if this fixes it, please? Thanks!

[social-warfare.zip](https://github.com/warfare-plugins/social-warfare/files/288889/social-warfare.zip)

**Comment by brianleejackson** [comment]
Awesome! You got it :+1: You can see it working here: [https://woorkup.com/wphostingspot-best-managed-wordpress-hosting/](https://woorkup.com/wphostingspot-best-managed-wordpress-hosting/)

Thanks.

**Comment by Nicholas-Cardot** [comment]
@brianleejackson Awesome. Thanks and sorry for the delay on this issue. It was a real stumper, and once I figured it out, I was kicking myself for not realizing the issue and having it solved right away.

Since you follow our GitHub, you might be interested in the article I published today on our blog. It's a look at our road map and goals for the 2.0 version that we're working on.

https://warfareplugins.com/future-social-warfare-wordpress-plugin/
